openedx/core/djangoapps/catalog/utils.py

```diff
@@ -469,7 +469,14 @@ def get_course_runs_for_course(course_uuid):  # lint-amnesty, pylint: disable=mi
         return []
 
 
-def get_owners_for_course(course_uuid):  # lint-amnesty, pylint: disable=missing-function-docstring
+def get_owners_for_course(course_uuid, traverse_pagination=True):
```

Comment:
you shouldnt need `traverse_pagination` as an kwarg - it should always be passed into `get_api_data` as `False` here because we're asking for a specific course. Also, this traversal was happening when the `course_uuid` was `None` and we've now added a guard `ValueError` exception.